common/scripts/configs/admiral.env.template

```diff
@@ -31,3 +31,4 @@ SHIPPABLE_HTTPS_PROXY=""
 SHIPPABLE_NO_PROXY=""
 ARCHITECTURE=""
 OPERATING_SYSTEM=""
+SWARM_DOCKER_VERSION=""
```

Comment:
just `DOCKER_VERSION` is good enough

Reply:
we are already loading  `DOCKER_VERSION` environment variable from `_installVersion` for arch/OS
https://github.com/Shippable/admiral/blob/master/common/scripts/x86_64/Ubuntu_14.04/_installVersion.env#L1

should i use `INSTALLED_DOCKER_VERSION` instead of this?